tests/dcerpc/dcerpc-dce-iface-03/test.rules

```diff
@@ -1 +1 @@
-alert tcp any any -> any any (msg:"DCE Iface test";flow:established,to_server;dce_iface:afa8bd80-7d8a-11c9-bef4-08002b102989,=1;sid:1;)
+alert tcp any any -> any any (msg:"DCE Iface test";flow:established,to_server;dcerpc.iface:afa8bd80-7d8a-11c9-bef4-08002b102989,=1;sid:1;)
```

Comment:
Could you please separate out the keyword changes in an existing test in a different commit?